http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/util/server/HttpsServer.java

```diff
@@ -75,7 +75,7 @@ public void start() {
             KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
             kmf.init(ks, ctPass);
 
-            sslContext = SSLContext.getInstance("TLSv1.2");
+            sslContext = SSLContext.getInstance("TLS");
```

Comment:
Why is this changed to TLS from TLSv1.2?

Reply:
To prevent this from being specific to a particular TLS version